### ISSUE TITLE
[BUG] Owned PRs appear and disappear

### ISSUE THREAD

**Issue by arielj** [fastruby/dash#56]
**Branch/Commit:**

Main branch

**Expected behavior:**

My owned PRs should always be displayed in the table.

**Actual behavior:**

Sometimes, a PR I have created is not in the list even though the PR is still open:

This PR for the stay updated project:
![image](https://user-images.githubusercontent.com/188464/101907827-dd135980-3b99-11eb-9225-bc5ba6cb2379.png)

Is not showing up here:
![image](https://user-images.githubusercontent.com/188464/101907889-f4524700-3b99-11eb-8e0e-253ea6b97b50.png)
(there are 2 PRs with the same name for other projects, but that should create any problem)

If I re-sync, the PR appears:
![image](https://user-images.githubusercontent.com/188464/101907954-151a9c80-3b9a-11eb-8651-528f2af84827.png)

All the PRs were there initially, for some reason some PRs get deleted.

**Steps to reproduce:**

I don't have specific steps to reproduce this, it's something the app does automatically. I don't have way to reproduce it consistently but it happened a few times the last few days.

**I will abide by the [code of conduct] (https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md)**

**Comment by cleicar** [comment]
@arielj do you think this story is duplicated with #59?

**Comment by arielj** [comment]
I don't think so, I was the owner of all those PRs but not assigned in any way, it was not a duplication issue but a PR not showing up (deleted for some reason, I think the scheduled job uses a user that may not have access to all my repos, and when I sync manually it uses my user so it has access to all my repos, not sure though)
